src/main/java/uk/gov/hmcts/reform/unspec/service/BusinessProcessService.java

```diff
@@ -1,31 +1,26 @@
 package uk.gov.hmcts.reform.unspec.service;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.unspec.callback.CaseEvent;
 import uk.gov.hmcts.reform.unspec.model.BusinessProcess;
 import uk.gov.hmcts.reform.unspec.model.CaseData;
 
-import java.util.List;
-import java.util.Map;
-
 import static uk.gov.hmcts.reform.unspec.enums.BusinessProcessStatus.READY;
 
 @Service
 @RequiredArgsConstructor
 public class BusinessProcessService {
 
-    private static final String ERROR_MESSAGE = "Business Process Error";
-
-    private final ObjectMapper mapper;
-
-    public List<String> updateBusinessProcess(Map<String, Object> data, CaseEvent caseEvent) {
-        CaseData caseData = mapper.convertValue(data, CaseData.class);
+    public CaseData updateBusinessProcess(CaseData caseData, CaseEvent caseEvent) {
```

Comment:
We still need to pass data map here, otherwise business process will not be set to READY

Reply:
good catch. done